consensus/validator.go

```diff
@@ -247,16 +255,18 @@ func (consensus *Consensus) onCommitted(msg *msg_pb.Message) {
 	if !consensus.isRightBlockNumCheck(recvMsg) {
 		return
 	}
+	if recvMsg.BlockNum > consensus.blockNum {
```

Comment:
the previous condition has "recvMsg.BlockNum-consensus.blockNum > consensusBlockNumBuffer" to avoid entering into sync mode prematurely. Now the buffer is gone, is the logic taking care of this case? "spinUpStateSync" won't disturb normal consensus logic?

Reply:
Removing buffer is to enable better sync consensus coordination, which spin sync immediately,  resulting in the benchmark illustrated in the PR description. For the previous condition, when will it happen when "entering into sync mode prematurely"? Could you specify?

Yes, `spinUpStateSync` does not interrupt message handling logic. The message will be added to FBFT log if passed aggregated signature verification.

Reply:
prematurely meaning it doesn't really miss the block, just that some messages are delivered out of order. Say committed_10 is delivered before committed_9, then it will trigger the sync, when it could self recover when the committed_9 is delivered. But as long as the sync won't interfere with consensus, it is fine to trigger sync anytime. Previously, I think when the node is in sync mode, it may interfere with consensus.

Reply:
Thanks for clarification. Yes, Sync will only used when deciding whether or not to sign a block during message handling. So shall be good here :)